test/ctia/entity/entities_test.clj

```diff
@@ -15,7 +17,8 @@
 #_
 (use-fixtures :once mth/fixture-schema-validation)
 
-(use-fixtures :each test-helpers/fixture-ctia-fast)
+(use-fixtures :once (join-fixtures [es-helpers/fixture-properties:es-store
```

Comment:
When I changed `:each` => `:once` it's because they only launch CTIA and/or only init some properties. The tests do not change these states, and with the `for-each-es-version`macro, the cleaning is delegated to the tests that really require it (while I doubled ES tests :-) )
So the purpose here is to limit as far as possible the actions only when it's necessary. With `:each`, in big ns tests we had fixtures applied for many tests that indeed do not require them.

> "it seems more robust to use :each for fixture-ctia"

why?

Reply:
> why?

In brief, IMO it encourages test isolation, which IMO makes our test suite more robust since it is independent of test ordering (or potential parallelism).

My reasoning is that it's possible that tests can interact with stateful `:once` fixtures since we're sharing a Trapperkeeper app with several deftests. Isolating tests from each other with `:each` means we can be more confident that our tests aren't passing by coincidence. We can also be more confident that trying different test runners or different test splitting techniques or refactoring tests will not break the test suite.